source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,102 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Date Operations
+~~~~~~~~~~~~~~~
+
+You can perform a date operation using the methods described in this
+section. Date operations either work on values of type ``MqlDate`` or
+return ``MqlDate`` values.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
```

Comment:
[suggestion]
The entries in this table do not seem to follow a consistent order paradigm. Since they are not all time-based (so we can't do smallest to largest), I recommend sorting them alphabetically. Since the Java Method and Aggregation Pipeline Operator do not always match, one would need to be picked for alphabetical. I'd recommend the Java Method since it is the first one readers' eyes see in the table.